apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/dashboard/HomeDashboard.jsx

```diff
@@ -399,36 +346,39 @@ function HomeDashboard() {
     const summaryInfo = [
         {
             title: 'Total APIs',
-            data: transform.formatNumberWithCommas(totalAPIs),
+            data: transform.formatNumberWithCommas((totalAPIs[totalAPIs.length-1] || 0)),
```

Comment:
this will fail in case the variable is undefined. Take ref. and fix the same, at all the places.
<img width="724" alt="Screenshot 2024-10-24 at 2 55 30 PM" src="https://github.com/user-attachments/assets/8f80972d-e0c2-49b7-b6c8-4f2d46332b41">

Reply:
`totalAPIs` will never be undefined or empty. Because we are already handling it inside the `observeFunc.setIssuesState` function.

Reply:
but I'll still add a undefined check